__tests__/integration.test.js

```diff
@@ -81,3 +82,36 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  it("GET:200 should send all articles", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles).toHaveLength(13);
+        articles.forEach((article) => {
+          expect(Object.keys(article)).toHaveLength(8);
```

Comment:
^ I wouldn't worry about this expect - as you may increase or decrease the amount of keys it has! :)

Reply:
Thanks for the feedback @nc-niamh-smith , I have removed the expect keys 😊